README.md

```diff
@@ -120,65 +92,55 @@ nwbfile.add_lab_meta_data(
     )
 )
 
-# Important: we add the fibers to the fibers table _after_ adding the metadata
-# This ensures that we can find this data in their tables of origin
-fibers_table.add_fiber(
-    excitation_source=0, #integers indicated rows of excitation sources table
-    photodetector=0,
-    fluorophores=[0], #potentially multiple fluorophores, so list of indices
-    location='my location',
-    notes='notes'
-)
-
-# Here we set up a list of fibers that our recording came from
-fibers_ref = DynamicTableRegion(
-    name="rois", 
-    data=[0], # potentially multiple fibers
-    description="source fibers", 
+# Create a raw FiberPhotometryResponseSeries, this is your main acquisition
+# We should create DynamicTableRegion referencing the correct rows for each table
+fiber_ref = DynamicTableRegion(
+    name="fiber",
+    description="source fiber",
+    data=[0],
     table=fibers_table
 )
-
-# Create a raw roiresponseseries, this is your main acquisition
-roi_response_series = RoiResponseSeries(
-    name="roi_response_series",
-    description="my roi response series",
-    data=np.random.randn(100, 1),
-    unit='F',
-    rate=30.0,
-    rois=fibers_ref,
-)
-
-# This is your processed data 
-deconv_roi_response_series = DeconvolvedRoiResponseSeries(
-    name="DeconvolvedRoiResponseSeries",
-    description="my roi response series",
+excitation_ref = DynamicTableRegion(
```

Comment:
It's a little clunky to have to specify a whole dynamic table region for each reference in the fiber photometry response series. It also forces the user to use `name="photodetector"` for the photodetector reference and so on, which can be confusing if you have multiple entries in each table that are named differently.

I think we should specify the dynamic table regions with integers or lists like we used to with the fibers table.